### PR TITLE
Update to imread-0.6.1

### PR DESCRIPTION
**Pull request by luispedro**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
The Travis CI builds appeared to be queued, but their status is not being shown on GitHub. Likely caused by [this ongoing issue]( https://www.traviscistatus.com/incidents/t68jk6rj92q4 ).

**Comment by luispedro** [comment]
Travis is all green now, so I think it should be ready to merge, but do let me know if anything is wrong.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ requirements:
     - python
     - setuptools
     - toolchain
-    - numpy x.x
+    - numpy 1.11.*
     - msinttypes  # [win]
     - jpeg 9.*
     - libpng >=1.6.28,<1.7
```
Comment (jakirkham) [inline recipe/meta.yaml]: This should now be `libpng >=1.6.32,<1.6.35`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ requirements:
 
   run:
     - python
-    - numpy x.x
+    - numpy >=1.11
     - jpeg 9.*
     - libpng >=1.6.28,<1.7
```
Comment (jakirkham) [inline recipe/meta.yaml]: This should also now be `libpng >=1.6.32,<1.6.35`.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -24,7 +24,7 @@ requirements:
     - python
     - setuptools
     - toolchain
-    - numpy x.x
+    - numpy 1.11.*
     - msinttypes  # [win]
     - jpeg 9.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please drop the `.` here.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -34,7 +34,7 @@ requirements:
 
   run:
     - python
-    - numpy x.x
+    - numpy >=1.11
     - jpeg 9.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also please drop the `.` here.

**Comment by jakirkham** [comment]
Also should now have `libtiff >=4.0.8,<4.0.10` and `zlib 1.2.11`. These are all taken from [this pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py ).

File: recipe/meta.yaml (hunk 3 above)
Comment (luispedro) [reply recipe/meta.yaml]: Do you mean `numpy 1.11*`?

File: recipe/meta.yaml (hunk 4 above)
Comment (luispedro) [reply recipe/meta.yaml]: For this one, I'm not sure what you mean.

**Comment by jakirkham** [comment]
`jpeg` is currently pinned as `9.*` in this recipe. However `jpeg` typically uses versions like `9a`and such. Hence it should be `9*` without the `.`.

**Pull request merged**